policyengine_us/variables/household/income/spm_unit/spm_unit_broadband_subsidy.py

```diff
@@ -0,0 +1,14 @@
+from policyengine_us.model_api import *
+
+
+class spm_unit_broadband_subsidy(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "SPM unit broadband subsidy"
```

Comment:
I think we can drop the entity from the label in future since it's duplicative

Reply:
Good point I'll merge now but we can do this for multiple variables together